### PR TITLE
Add Specification Committee and PMC Role descriptions

### PR DESCRIPTION
**Pull request by waynebeaton**
Signed-off-by: Wayne Beaton <wayne.beaton@eclipse-foundation.org>

### CODE REVIEW

File: operations.adoc [hunk 1]
```diff
@@ -6,6 +6,32 @@ Changes to this document must be approved by a simple majority vote of the Jakar
 
 This is a public document.
 
+== Specification Committee
+
+The Specification Committee is responsible for implementing the Eclipse Foundation Specification Process (EFSP) for all Specification Projects (as that term is defined by the EFSP). This includes the adaptation of the EFSP for Jakarta which is referred to as the Jakarta EE Specification Process (JESP).
```
Comment (kwsutter) [inline operations.adoc]: This position has been in the back of mind for quite some time...  Why does the Jakarta EE Spec Committee define the EFSP for all of the projects within the Eclipse Foundation?  I understand why we did that originally.  But, now that it is "complete", it seems that the EFSP should be governed by an independent group outside of the Jakarta EE working group.

File: operations.adoc [hunk 2]
```diff
@@ -6,6 +6,32 @@ Changes to this document must be approved by a simple majority vote of the Jakar
...
+
+The Specification Committee is responsible for producing, publishing and maintaining operational guidance documentation for specification projects. This includes the minimum requirements and process for producing a Final Specification. It also includes operational guidance for running a specifications TCK for the purpose of testing for compatibility.
+
+The Specification Chair (or their delegate) is responsible for initiating ballots, tallying their results, disseminating them to the community, and (when appropriate; e.g., in the case a release review ballot) reporting them to the EMO.
+
+The Specification Committee is also responsible for defining and refining how they implement the EFSP/JESP.
```
Comment (kwsutter) [inline operations.adoc]: Follow up on my previous comment...  Whether the Jakarta EE Working Group should be responsible for defining the EFSP for all working groups...

File: operations.adoc [hunk 3]
```diff
@@ -6,6 +6,32 @@ Changes to this document must be approved by a simple majority vote of the Jakar
...
+
+The Specification Committee is not responsible for the day-to-day operation of Specification Projects, and do not have any responsibility or authority regarding the structure or organization of Specification Projects and corresponding Top Level Project.
+ 
+== Project Management Committee (PMC)
+
+The primary role of the PMC is to ensure that project teams are implementing the Eclipse Development Process. In particular, the PMC monitors project activity to ensure that project teams are operating in an open and transparent manner. The PMC reviews and approves (or vetos) committer elections, first validating that candidate committers have demonstrated sufficient merit. 
+
+The PMC is further responsible for ensuring that project teams abiding by the Eclipse IP Policy and implementing the IP Due Diligence process. In practical terms, the PMC approves intellectual property contributions (CQs) and provides (limited) guidance to project teams regarding intellectual property (deferring to the Eclipse IP Team as necessary).
+
+The PMC is responsible for the structure of the TLP and the organization of the open source (software and Specification) projects contained within.
```
Comment (kwsutter) [inline operations.adoc]: TLP == Top Level Project?

File: operations.adoc (hunk 2 above)
Comment (waynebeaton) [reply operations.adoc]: Again, the intent is that the Specification Committee defines and refines how **they implement** the EFSP.

At least in part, the problem here is that I was trying to draft this in a generic way so that I could reuse it for the other working groups that are starting up specification processes.

**Comment by waynebeaton** [comment]
> I would like to have a discussion on whether the Jakarta EE Working Group should continue to own and define the overall EFSP.

I believe that I've clarified this in other comments, but for completeness, the Jakarta EE Working Group Specification Committee does **not** own the EFSP.

File: operations.adoc [hunk 4]
```diff
@@ -6,6 +6,32 @@ Changes to this document must be approved by a simple majority vote of the Jakar
...
+
+The Specification Committee is responsible for producing, publishing and maintaining operational guidance documentation for specification projects. This includes the minimum requirements and process for producing a Final Specification. It also includes operational guidance for running a specifications TCK for the purpose of testing for compatibility.
+
+The Specification Chair (or their delegate) is responsible for initiating ballots, tallying their results, disseminating them to the community, and (when appropriate; e.g., in the case a release review ballot) reporting them to the EMO.
+
+The Specification Committee is also responsible for defining and refining how they implement the EFSP/JESP for those Specification Projects under their purview.
+
+The Specification Committee is not responsible for the day-to-day operation of Specification Projects, and do not have any responsibility or authority regarding the structure or organization of Specification Projects and corresponding Top-Level Project.
```
Comment (dblevins) [inline operations.adoc]: We should discuss the intent and maybe rephrase.  We're ideally not involved in the day to day, but we definitely have imposed some structure requirements on projects such as they need to have an issue tracker for accepting TCK challenges, specific labels that must be in the issue tracker, specific groupIds that must be used.  We favor giving projects flexibility and making recommendations vs rules, but the specification committee does impose (ideally minimal) structure.

Comment (waynebeaton) [reply operations.adoc]: Maybe it's enough to remove the "not responsible" paragraph?
